src/main/scala/Main.scala

```diff
@@ -1,7 +1,36 @@
 import scala.annotation.tailrec
-
+//AK
 object Main {
   def main(args:Array[String]): Unit ={
+    val input = scala.io.StdIn.readLine()
+    val resultA = buildString("",0,input,1)
 
   }
+  val numbers:Array[(String,String)] = Array(Tuple2("",""),
+    Tuple2(" thousands and "," thousands and "),
+    Tuple2(" million "," million "),
+    Tuple2(" billion "," milliard "),
+    Tuple2(" trillion "," billion "),
+    Tuple2(" quadrillion "," billiard "),
+    Tuple2(" quintillion "," trillion "),
+    Tuple2(" sextillion "," trilliard ")
+  )
+  def getPosition(index:Int, shortOrLong:Int): String ={
+    shortOrLong match{
+      case 1 => numbers(index)._1
+      case 2 => numbers(index)._2
+      case _ => ""
+    }}
+  @tailrec
+  def buildString(currentString:String,byThrees:Int,message:String,printExample:Int): String ={
+    if (message.length > 3) {
```

Comment:
you keep getting the message length, why not just have a val for it instead of trying to get it all the time?